packages/manager/modules/support/package.json

```diff
@@ -21,7 +21,8 @@
     "@ovh-ux/component-rollup-config": "^5.1.0"
   },
   "peerDependencies": {
-    "@ovh-ux/manager-core": "^6.0.1",
+    "@ovh-ux/manager-config": "^0.3.0",
```

Comment:
Is `@ovh-ux/manager-config` directly used in support module ?

Reply:
nope! removed! thanks!